lib/dogma.ex

```diff
@@ -6,20 +6,29 @@ defmodule Dogma do
   other modules through the divine `run/3` function.
   """
 
-  alias Dogma.Formatter
   alias Dogma.Rules
   alias Dogma.ScriptSources
 
   @version Mix.Project.config[:version]
 
-  def run(dir, config, formatter) do
+  def run(dir, config, events) do
```

Comment:
Can we call this `gen_event` or `event_listener` or something like that please? :)